### ISSUE TITLE
Election related stuff

### ISSUE THREAD

**Issue by jeinarsson** [demokratikollen/demokratikollen#35]
Maps like
http://en.wikipedia.org/wiki/File:SwedishGeneralElection2010-map-SwedenDemocrats.svg
for the different parties could be a nice thing if we have "party pages". Or regional pages for different parts of sweden. Bring out your GIS nerds!

Official stats here
http://www.val.se/val/val2014/statistik/index.html (Also shapefiles and other GIS-stuff!)

Valdeltagande (and much more) (election 2014 stats during winter, could be good to be prepared for it)
http://www.scb.se/sv_/Hitta-statistik/Statistik-efter-amne/Demokrati/Allmanna-val/Allmanna-val-valdeltagandeundersokningen/12304/2010A01X/

I also ordered every parliament election "VALU" from 1991 in excel format. Let's see what happens!

**Comment by joelgoop** [comment]
I have started a little bit on party pages (nothing pushed yet), and I have converted SCB's municipality shapefile to [topojson](https://github.com/mbostock/topojson) and put it in the btsync. From the topojson it will hopefully be easy to plot with d3.

**Comment by joelgoop** [comment]
A map of the 2014 election is now drawn on each party page (/partierna/<abbr>.html) and mouseover on a municipality shows a history of election results for that particular area. Merged into develop as of fabc5d9b5f327642bbafbb92f752751f4223f711.

**Comment by rasmuse** [comment]
I did some alpha testing with our friends P&S yesterday. Some feedback:
- Map is appreciated. Testers find it interesting to look at historical data for municipalities.
- I dropped the idea about possibly showing additional diagrams on the side of the map, which was appreciated. This may however be hard to design well for very small screens (e.g. mobile phone), because the additional diagrams end up below the map?! However, the testers agreed that a tooltip close to the clicked/pushed municipality is a very good solution that should remain.
- Testers both found it very confusing that the vertical axis in historical results does not start on zero. They realize the complication that say a 10-point change from 50% to 40% on a big party will not look as dramatic as it is, but they both agreed that the pros of starting the vertical axis on zero outweigh the cons.
- Contrast is too low according to both testers, especially in Sverigedemokraterna figure (yellow). The blue and red colors worked best in terms of contrast. Perhaps we could mix some black into the upper end of the color scales to increase contrast? The brightest end of the color scales should not be made brighter.
- The size of the map is problematic:
  - First, the maximal zoom level allowed on the iPad (Safari) was not enough to be able to hit the desired municipality on the touch display, at least not small municipalities.
  - Municipality size and tooltip size do not have a reasonable relation: At the zoom level where testers could clearly distinguish the municipalities and hit them on the touch display, the tooltip would no longer fit on the screen. Suggestion: Make the map a bit bigger and reduce padding in the tooltips.
